src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -122,35 +102,56 @@ export default App.extend({
   onStart(options, collection) {
     this.collection = collection;
     this.filteredCollection = collection.clone();
+    this.editableCollection = collection.clone();
 
+    this.listenTo(this.filteredCollection, 'reset', this.showCountView);
+    this.showCountView();
+
+    this.listenTo(this.editableCollection, 'reset', this.toggleBulkSelect);
+    this.toggleBulkSelect();
+
+    this.showList();
+  },
+  // NOTE: Shows views dependent on getState().getType()
+  showTypeViews() {
+    this.showListTitle();
+    this.showTypeToggleView();
+    this.showDateFilter();
+    this.showSortDroplist();
+    this.showTableHeaders();
+  },
+  showList() {
     const collectionView = new ListView({
       collection: this.collection,
       state: this.getState(),
       viewComparator: this.getComparator(),
     });
 
-    this.listenTo(collectionView, 'filtered', filtered => {
-      this.filteredCollection.reset(filtered);
-      this.toggleBulkSelect();
-      this.showCountView();
-    });
-
-    this.listenTo(collectionView, 'click:patientSidebarButton', ({ model }) => {
-      const patient = model.getPatient();
-      Radio.request('sidebar', 'start', 'patient', { patient });
+    this.listenTo(collectionView, {
+      'filtered'(filtered) {
+        this.filteredCollection.reset(filtered);
+        this.editableCollection.reset(this._getListEditable(collectionView));
+      },
+      'change:canEdit'() {
+        this.editableCollection.reset(this._getListEditable(collectionView));
```

Comment:
There's the potential where an action/flow can be selected and then switched to `canEdit === false`. When that currently happens, the action/flow still has the `is-selected` classname when it shouldn't.

When `change:canEdit` is triggered, looks like that action/flow is still being stored in `state.actionsSelected`/`state.flowsSelected`.

Everything else with multi-select works well. Only thing I noticed.

Related screen recording:

https://github.com/RoundingWell/care-ops-frontend/assets/35355575/130e566c-8f7c-4f9b-8cd2-80ca472a980e

Reply:
so the `getSelected` on the state is already dealing with filtering out actions/flows that aren't in the editableCollection..  I _think_ this was so we didn't have to manage `actionSelected`/`flowsSelected` as users did the find in list.. _so_ I think we can just handle this visually on that view

Reply:
I haven't had time to test this fully, but I think the latest commit is the solution